calitp/storage.py

```diff
@@ -672,9 +565,10 @@ def download_feed(
                 cls=GTFSFeedExtractInfo,
                 fs=get_fs(),
                 partitions={
-                    "dt": pendulum.today().date(),
+                    "dt": pendulum.yesterday().date(),
```

Comment:
isn't this already used by RT in v3 "prod"? what effect will this change have / why change from today to yesterday?

Reply:
This is just for local testing, it's not actually used anywhere